deploy/docs/v1_migration_doc.md

```diff
@@ -0,0 +1,178 @@
+- The Fluentd deployments have been changed to **statefulsets** to support the use of **persistent volumes**. This will allow better buffering behavior. They also now include `“fluentd”` in their names. This is a breaking change for non-Helm users as the deployments will not be cleaned up upon upgrade, leading to duplicate events (logs and metrics will not experience data duplication).
+- The unified Fluentd `statefulsets` have been split into set of two different Fluentd's, one for `logs` and the other one for `metrics`.
+- We now support the collection of renamed metrics (for Kubernetes version `1.17+`).
+### How to upgrade for Non-helm Users
+#### 1. Tear down existing Fluentd, Prometheus, Fluentbit resources
+Run the commands mentioned [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/Non_Helm_Installation.md#tear-down) to delete the respective resources.
+
+#### 2. Deploy Fluentd, Fluentbit and Prometheus again with the version 1.0.0 yaml
+Follow the below steps to deploy new resources. 
+##### 2.1 Deploy Fluentd
+- Non-Helm users who have made changes to configs in the [environment variable sections](#https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/release-v0.17/deploy/kubernetes/fluentd-sumologic.yaml.tmpl#L627-L678) of the `fluentd-sumologic.yaml` file will need to move those config changes directly into the Fluentd pipeline.
+
+- Run the below command to get the `fluentd-sumologic.yaml` manifest for version `v1.0.0` and  then make the changes identified in the above step.
+```bash
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v0.17/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
+sed 's/\$NAMESPACE'"/<NAMESPACE>/g" | \
+sed 's/cluster kubernetes/cluster <CLUSTER_NAME>/g'  >> fluentd-sumologic.yaml
+```
+
+- Non-Helm users running a Kubernetes version of 1.13 or older will need to **remove the following filter plugin section from their Fluentd pipeline. This is required to prevent data duplication.**
+```yaml
+<filter prometheus.metrics**> # NOTE: Remove this filter if you are running Kubernetes 1.13 or below.
+  @type grep
+  <exclude>
+    key @metric
+    pattern /^apiserver_request_count|^apiserver_request_latencies_summary|^kubelet_runtime_operations_latency_microseconds|^kubelet_docker_operations_latency_microseconds|^kubelet_docker_operations_errors$/
+  </exclude>
+</filter>
+```
+##### 2.2 Deploy Prometheus
+- Follow steps mentioned [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/Non_Helm_Installation.md#deploy-prometheus) to deploy prometheus.
+##### 2.3: Deploy Fluentbit
+- Follow steps mentioned [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/Non_Helm_Installation.md#deploy-fluentbit) to deploy fluent-bit.
```

Comment:
I believe the proper name is Fluent Bit, we should make sure we call it properly.